manifests/init.pp

```diff
@@ -46,6 +46,9 @@
 #
 # $qpid_router_broker_port::            Port of qpidd broker to connect to
 #
+# $enable_ostree::                      Boolean to enable ostree plugin. This requires existence of a ostree install.Defaults to false
```

Comment:
Mention of defaults not needed as that is handled by the value in the params file